sonar-xml-plugin/pom.xml

```diff
@@ -147,8 +140,8 @@
             <configuration>
               <rules>
                 <requireFilesSize>
-                  <maxsize>1500000</maxsize>
-                  <minsize>1400000</minsize>
+                  <maxsize>2000000</maxsize>
```

Comment:
Size increase is due to woodstox. It was shaded before because xml-parsing from commons didn't force this library (and now it does https://github.com/SonarSource/sonar-analyzer-commons/commit/80ef751c615a298e90929865bf09f90e61c068b7#diff-72c9d550fb8018fc48eb489aef6e8170R197).